README.md

```diff
@@ -91,9 +141,24 @@ The following table gives the a description of the fields and corresponding valu
 
 #### Copy Number Variants (CNVs)
 
-Multiple callers were used to determine copy number variants (CNVs). A consensus CNV file was constructed, consisting of CNVs that were called by 2 or more copy number callers: ControlFreeC, CNVkit, and GATK CNV. See https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc-consensus-calling.md for more detail on how the calls were combined. The various nomenclatures used by the different callers are harmonized to standard descriptions of CNVs: deep deletion for when copy number equals zero, loss for when copy number is less than ploidy, neutral if copy number is the same as the genome copy number (cancer genomes may have a ploidy other than two), gain for up to two times ploidy, and amplification for a gain of more than two times ploidy. Copy number variants were retained only if they overlapped by at least 1 base pair with a gene’s exon. Further annotations, including alternative gene and protein IDs and cancer references, were also added.
+Multiple callers were used to determine copy number variants (CNVs). 
+A consensus CNV file was constructed, consisting of CNVs that were called by 2 or more copy number callers: ControlFreeC, CNVkit, and GATK CNV. 
```

Comment:
I do not believe that our modules used the consensus using GATK CNV for tables, but let's check with @ewafula - we may just update this for v12 (as in, it is not in the release using GATK CNV)

Reply:
Eric confirmed that this was correct on Slack